tupaia-ci-cd.Dockerfile

```diff
@@ -15,69 +20,47 @@ COPY package.json ./
 COPY yarn.lock ./
 COPY babel.config.json ./
 COPY .babelrc-ts.js ./
+COPY tsconfig-js.json ./
 RUN mkdir ./scripts
 COPY scripts/. ./scripts
 
+RUN mkdir -p ./packages/devops/scripts/ci
+COPY packages/devops/scripts/ci/makePlaceholderPackageFolders.sh ./packages/devops/scripts/ci
+
+RUN ./packages/devops/scripts/ci/makePlaceholderPackageFolders.sh
```

Comment:
Minor: I like that this makes the Dockerfile shorter but it comes at the cost of no longer colocating the `mkdir` and `COPY`. My personal preference is colocation to make changes easier and a long Dockerfile, because it's not hiding the root cause, which is that we have a fat image. Fat images are bad (generally speaking in container land), and the solution is to split it out. Or if we decide to have a fat image we should embrace it :)

Reply:
Yeah no worries, I'm happy to change this back. There was another use for the util at one point during development but that disappeared, so there's less justification for it now